consensus/safety-rules/src/serializer.rs

```diff
@@ -118,3 +118,24 @@ impl TSerializerClient for LocalService {
             .handle_message(input_message)
     }
 }
+
+#[cfg(any(test, feature = "fuzzing"))]
+/// This module allows the fuzzer to fuzz the handle_message() method exposed by the serializer
+/// service.
+pub mod fuzzing {
+    use crate::{serializer::SafetyRulesInput, test_utils, Error};
+
+    pub fn fuzz_handle_message(safety_rules_input: SafetyRulesInput) -> Result<Vec<u8>, Error> {
```

Comment:
Same, I'm not a big fan of seeing fuzz code like this :/

Reply:
Merged into fuzzing_utils.rs. See above!